testing/tests/DevExpress.ui.widgets.editors/autocomplete.tests.js

```diff
@@ -1469,6 +1469,49 @@ QUnit.module('regressions', {
         assert.strictEqual(selectionChangedStub.lastCall.args[0].selectedItem, null);
     });
 
+    QUnit.test('onSelectionChanged event should trigger on item selection if its text is equal to input text (T991350)', function(assert) {
+        const selectionChangedStub = sinon.stub();
+        const timeToWait = 300;
+
+        this.instance.option({
+            searchTimeout: timeToWait,
+            onSelectionChanged: selectionChangedStub
+        });
+
+        this.keyboard
+            .type('item 2');
+
+        this.clock.tick(timeToWait + 100);
+        const $listItems = $(this.instance.content()).find(`.${LIST_ITEM_CLASS}`);
```

Comment:
```suggestion
        const $listItems = this.instance.$content().find(`.${LIST_ITEM_CLASS}`);
```

Reply:
we do not have $content() for Autocomplete